src/Services/GTT/shared/GTT.Infrastructure/Repositories/CommunityRepository.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Dapper;
+using GTT.Application;
+using GTT.Application.Interfaces.Repositories;
+using GTT.Application.Response;
+using System.Data;
+
+namespace GTT.Infrastructure.Repositories
+{
+    public class CommunityRepository : ICommunityRepository
+    {
+        private readonly IDbConnection _connection;
+
+        public CommunityRepository(IDbConnectionFactory dbConnectionFactory)
+        {
+            _connection = dbConnectionFactory.CreateConnection();
+        }
+        public CommunityRepository(IDbConnection dbConnection)
+        {
+            _connection = dbConnection;
+        }
+
+        public async Task<BaseResponseModel> GetAllCommunity()
+        {
+            try
+            {
+                var getAllSql = @"SELECT CommunityId
+                              ,CommunityName
+                              ,Image
```

Comment:
Why image exist here? I think it should call is icon, because image will return URL, icon return svg. If you set it is image, this make some confuse.